ddsrouter_cmake/cmake/project/read_branch_commit_hash.cmake

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###############################################################################
+# Commit hash Reader macro
+###############################################################################
+
```

Comment:
Remove blank lines

Reply:
same space existed in `read_version.cmake`, but removed

Reply:
NIT: It is not the same space, neither it has been removed.
However, we do not follow any CMake specific format, so it is not relevant, it was just a small suggestions

Reply:
Yes, it has been removed, but file name has changed to `read_commit_hash.cmake` to remove *branch*